pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -250,6 +251,23 @@ func NewController(kubeClient kubelib.Client, options Options) *Controller {
 	}
 
 	c.namespaces = kclient.New[*v1.Namespace](kubeClient)
+
+	if features.EnableAmbientControllers {
```

Comment:
What is this? seems likely a bad merge? I think this was removed in some unrelated PR recently

Reply:
still present in master https://github.com/istio/istio/blob/master/pilot/pkg/features/pilot.go#L550 ?

Reply:
I don't mean this flag I mean L255-L270. Why would adding WE make us need this now? It was explicitly removed